Makefile

```diff
@@ -51,8 +51,16 @@ else
 	BUILDER_IMAGE_TAG := $(shell git log -1 --pretty=%h $(BUILDER_IMAGE_DOCKERFILE))
 endif
 
+ifeq ($(shell command -v docker 2> /dev/null),)
+    BUILDER=podman
+    IMAGE_TOOL=podamn
```

Comment:
How about renaming it to `CONTAINER_RUNTIME` as it does not only pull/push images?